vassal-app/src/main/java/VASSAL/counters/ReturnToDeck.java

```diff
@@ -50,19 +51,16 @@
 import javax.swing.JDialog;
 import javax.swing.JLabel;
 import javax.swing.JList;
-import javax.swing.JPanel;
 import javax.swing.JTextField;
 import javax.swing.KeyStroke;
 import javax.swing.ListSelectionModel;
-import javax.swing.SwingUtilities;
-
-import net.miginfocom.swing.MigLayout;
 
 /**
  * GamePiece trait that returns a piece to a {@link DrawPile}
  */
 public class ReturnToDeck extends Decorator implements TranslatablePiece {
-  public static final String ID = "return;"; // NON-NLS
+  public static final String OLD_ID = "return;"; // NON-NLS
+  public static final String ID = "todeck;"; // NON-NLS
```

Comment:
This is one way to do to it, but it's not really needed unless the existing State format can't be extended. I would add a new State variables version_number at the end of the existing State. This will default to blank in the old version and you will set to 2 in the new version. Then, follow it with any new State variables you need. You can then handle editing and processing based on the Version number.

Reply:
Okay I think the new version tightens this up (and fixes the configurer problem above)